panel/widgets/chatbox.py

```diff
@@ -611,5 +612,37 @@ def clear(self) -> None:
         """
         self.value = []
 
+    def _serialize_message(self, message: Union[List[Any], Any]) -> str:
+        if isinstance(message, Iterable) and not isinstance(message, str):
+            return "\n".join(self._serialize_message(m) for m in message)
+
+        if hasattr(message, "value"):
+            message = message.value
+        elif hasattr(message, "object"):
+            message = message.object
+        message = str(message)
+        return message
+
+    def export(self, serialize: bool = True) -> List[Dict[str, Union[List[Any], Any]]]:
```

Comment:
I think we should add `format` to this and then let users select between OpenAI, tuple and the old dictionary format.